UnitsNet/UnitDefinitions/Molarity.json

```diff
@@ -0,0 +1,34 @@
+﻿{
+  "Name": "Molarity",
+  "BaseUnit": "MolesPerCubicMeter",
+  "XmlDoc": "Molar concentration, also called molarity, amount concentration or substance concentration, is a measure of the concentration of a solute in a solution, or of any chemical species, in terms of amount of substance in a given volume. ",
+  "XmlDocRemarks": "https://en.wikipedia.org/wiki/Molar_concentration",
+  "Units": [
+    {
+      "SingularName": "MolesPerCubicMeter",
+      "PluralName": "MolesPerCubicMeter",
+      "FromUnitToBaseFunc": "x",
+      "FromBaseToUnitFunc": "x",
+      "Prefixes": [],
+      "Localization": [
+        {
+          "Culture": "en-US",
+          "Abbreviations": [ "mol/m³" ]
+        }
+      ]
+    },
+    {
+      "SingularName": "MolesPerLitre",
```

Comment:
I think we should name this `MolesPerLiter`, as we are using international/US English and it would be consistent with `Volume.Liters`.